app/components/team_sidebar/team_list/team_item/team_icon.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
The style container can be deleted.

Reply:
Both container and containerSelected are used. Right now I think it is simpler to leave the as they are instead of trying to merge them into a single one.